deploy/charts/cert-manager/values.yaml

```diff
@@ -23,6 +23,7 @@ strategy: {}
 
 image:
   repository: quay.io/jetstack/cert-manager-controller
+  injectorRepository: quay.io/jetstack/cert-manager-injectorcontroller
```

Comment:
I think I'm going to follow up to this PR to move it into its own chart, which should also make this syntax a little more consistent, i.e. we can have:

```
injector:
  image:
    repository: ..
```

etc 😄